src/SelfAssessment/EmergencyAssessment.js

```diff
@@ -64,9 +64,9 @@ const ChoiceButtons = ({ agreePress, disagreePress }) => {
         accessibilityRole="button"
         style={style.button}
       >
-        <GlobalText style={{ ...style.buttonText, ...style.boldText }}>
+        <GlobalText style={[style.buttonText, style.boldText]}>
```

Comment:
Just curious if this change was due a pattern we are using or is it better somehow?